src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/zipfilestatus/ZipFileStatusServiceTest.java

```diff
@@ -80,4 +110,20 @@ public void should_return_empty_lists_when_no_data_for_zip_file_was_found() {
         assertThat(result.envelopes).isNotNull().isEmpty();
         assertThat(result.events).isNotNull().isEmpty();
     }
+
+    private Envelope envelope(UUID id, String container, Status status) {
+        Envelope envelope = mock(Envelope.class);
+        given(envelope.getId()).willReturn(id);
```

Comment:
using mocks becuase there is no setter for ID, plus we only use a couple of fields from this big model